e2e/charts/ksm/values.yaml

```diff
@@ -0,0 +1 @@
+kube-state-metrics: {}
```

Comment:
I am a bit confused, why adding a new chart with single dependency for ksm?
Cannot we  deploy directly ksm chart?

Reply:
I would have understood if it was a dependency of the scraper to place it there 😃

Reply:
The scraper needs ksm to be there to scrape it, but now that the scraper is part of the `e2e-resources` chart perhaps we could add ksm as a dependency from the latter. WDYT?

Reply:
exactly!